lte.js

```diff
@@ -11,19 +11,22 @@ const __ = require('./_internal/placeholder')
  *
  * @synopsis
  * ```coffeescript [specscript]
- * var value any,
- *   leftCompare any,
- *   rightCompare any,
- *   left (value=>Promise|leftCompare)|leftCompare,
- *   right (value=>Promise|rightCompare)|rightCompare
- *
- * lte(left, right)(value) -> Promise|boolean
+ * lte(leftValue any, rightValue any) -> boolean
+ * lte(leftValue any, right function)(value any) -> Promise|boolean
+ * lte(left function, rightValue any)(value any) -> Promise|boolean
+ * lte(left function, right function)(value any) -> Promise|boolean
  * ```
  *
  * @description
  * Test if a left value is less than or equal (`<=`) to a right value. Either parameter may be an actual value.
  *
```

Comment:
let's add here

If both arguments are values, `lte` eagerly computes and returns a boolean value.